app/tasks/raster_tile_cache_assets/raster_tile_cache_assets.py

```diff
@@ -150,7 +150,9 @@ async def raster_tile_cache_asset(
         symbology_jobs: List[Job]
         symbology_uri: str
 
-        symbology_co = source_asset_co.copy(deep=True)
+        symbology_co = source_asset_co.copy(
```

Comment:
Could you add a comment or move this out into a function with a clearer name of what's happening? Had to do dig through the  code to understand why this made sense.

Reply:
Will do.